mlx/traceability/directives/item_pie_chart_directive.py

```diff
@@ -399,6 +399,31 @@ def _get_explode_values(labels, label_set):
             explode[uncovered_index] = 0.05
         return explode
 
+    def _determine_headings(self, app):
```

Comment:
This seems unrelated. Or is there a reason?

Reply:
These were meant for https://github.com/melexis/sphinx-traceability-extension/pull/379 but I had already created a new branch